### ISSUE TITLE
Css and Js files are not found anymore.

### ISSUE THREAD

**Issue by garyee** [gitlist-php/gitlist#46]
Hi,

so after finding a solution to problem #45.
I got my gitlist as plain html ... with no js and css.
When I call one css directly I get the following:

```
NotFoundHttpException in RouterListener.php line 125: No route found for "GET /themes/default/css/style.css"

    in RouterListener.php line 125
    at RouterListener->onKernelRequest(object(GetResponseEvent), 'kernel.request', object(EventDispatcher))
    at call_user_func(array(object(RouterListener), 'onKernelRequest'), object(GetResponseEvent), 'kernel.request', object(EventDispatcher)) in EventDispatcher.php line 212
    at EventDispatcher->doDispatch(array(array(object(RouterListener), 'onKernelRequest'), array(object(RouterListener), 'onKernelRequest'), array(object(MiddlewareListener), 'onKernelRequest')), 'kernel.request', object(GetResponseEvent)) in EventDispatcher.php line 44
    at EventDispatcher->dispatch('kernel.request', object(GetResponseEvent)) in HttpKernel.php line 129
    at HttpKernel->handleRaw(object(Request), '1') in HttpKernel.php line 68
    at HttpKernel->handle(object(Request), '1', true) in Application.php line 496
    at Application->handle(object(Request)) in Application.php line 477
    at Application->run() in index.php line 29
```

**Comment by alehaa** [comment]
Hi @garyee,

there's no active development in this fork anymore, as most of the changes have been merged upstream (see #42). Can you check, if this bug also affects upstream release v.0.6?

**Comment by garyee** [comment]
no got fixed with v.0.6
